src/StudyRegistration/VLMDSubmission/CDESubmission.tsx

```diff
@@ -0,0 +1,305 @@
+    switch (formSubmissionStatus.status) {
+    case 'success':
+      return (
+        <div className='vlmd-sub-container'>
+          <div className='study-reg-form-container'>
+            <Result
+              status={formSubmissionStatus.status}
+              title='Your CDE has been submitted!'
+              subTitle='Thank you for your submission!'
+              extra={[
+                <Link key='discovery' to={'/discovery'}>
+                  <Button>Go To Discovery Page</Button>
+                </Link>,
+              ]}
+            />
+          </div>
+        </div>
+      );
+    case 'error':
+      return (
+        <div className='vlmd-sub-container'>
+          <div className='study-reg-form-container'>
+            <Result
+              status={formSubmissionStatus.status}
+              title='A problem has occurred during submission!'
+              subTitle={formSubmissionStatus.text}
+              extra={[
+                <Button type='primary' key='close' onClick={() => { setCDEFormSubmitting(false); setFormSubmissionStatus(null); }}>
+                    Close
+                </Button>,
+              ]}
+            />
+          </div>
+        </div>
+      );
+    default:
+      return null;
+    }
+  }
+
+  return (
+    <div className='vlmd-sub-container'>
+      <div className='study-reg-form-container'>
+        <Form
+          className='study-reg-form'
+          {...FORM_LAYOUT}
+          form={cdeForm}
+          name='cde-sub-form'
+          disabled={props.disableCDESubmissionForm}
+          onFinish={onFinish}
+          onFinishFailed={() => { setCDEFormSubmitting(false); }}
+          validateMessages={VALIDATE_MESSAGE}
+          onValuesChange={(changedValues) => {
+            if (Object.keys(changedValues).includes('coreCDEs')) {
+              const changedCoreCDECheckboxValues = changedValues.coreCDEs;
+              setSelectedCoreCDEs(changedCoreCDECheckboxValues);
+            }
+            if (Object.keys(changedValues).includes('selectedCDEs')) {
+              const changedCDESelectValues = changedValues.selectedCDEs;
+              const updatedCoreCDESelectValues: any[] = [];
+              const updatedNonCoreCDEselectValues: any[] = [];
+              changedCDESelectValues.forEach((element) => {
+                if (cdeInfoFromMDS.findIndex((entry) => (entry.option === element && entry.isCoreCDE)) !== -1) {
+                  updatedCoreCDESelectValues.push(element);
+                } else {
+                  updatedNonCoreCDEselectValues.push(element);
+                }
+              });
+              setSelectedCoreCDEs(updatedCoreCDESelectValues);
+              setSelectedNonCoreCDEs(updatedNonCoreCDEselectValues);
+            }
+          }}
+        >
+          <Divider plain>HEAL CDEs</Divider>
+          {(props.disableCDESubmissionForm) ? (
+            <Typography style={{ textAlign: 'center' }}>
+              (PLACEHOLDER) This form has been disabled
```

Comment:
Question: 

Do we want this to say (PLACEHOLDER) ?

Reply:
good catch! updated